client/src/pages/CreateContest/CreateContest.tsx

```diff
@@ -19,29 +22,32 @@ export default function CreateContest():JSX.Element {
       images
     }: Contest
   ) => {
-    const contest = {
-      title: title,
-      description: description,
-      prizeAmount: prizeAmount,
-      deadlineDate: deadlineDate,
-      images: images,
-    };
+    const contest = { title, description, prizeAmount, deadlineDate, images };
 
-    addContest(contest);
+    addContest(contest)
+      .then((res) => {
+        if (!res.error) {
```

Comment:
Maybe update the snack bar accordingly for successful contest creation/error handling?